common/display/displayqueue.cpp

```diff
@@ -64,7 +64,9 @@ DisplayQueue::~DisplayQueue() {
 }
 
 bool DisplayQueue::Initialize(uint32_t pipe, uint32_t width, uint32_t height,
+                              FrameBufferManager *pFBManager,
                               DisplayPlaneHandler* plane_handler) {
+  pFBuffer_manager_ = pFBManager;
```

Comment:
just have this and every where else frame_buffer_manager_ and change pFBManager to fbmanager :)

Reply:
frame_buffer_manager_ or frame_buffer_manager?